frontend/src/cljs/imas_seamap/views.cljs

```diff
@@ -646,23 +649,68 @@
+                                       :tabs
+                                       [{:id         "org"
+                                         :title      "By Organisation"
+                                         :categories [:organisation :data_classification]}
+                                        {:id         "cat"
+                                         :title      "By Category"
+                                         :categories [:data_classification]}]}])}]
      [b/button
       {:icon     "media"
        :text     "Imagery Layers"
-       :on-click #(re-frame/dispatch [:drawer-panel-stack/push :drawer-panel/layer-panel {:group :imagery :title "Imagery Layers"}])}]
+       :on-click #(re-frame/dispatch [:drawer-panel-stack/push :drawer-panel/catalogue-layers
+                                      {:group :imagery
+                                       :title "Imagery Layers"
+                                       :tabs
+                                       [{:id         "org"
+                                         :title      "By Organisation"
+                                         :categories [:organisation :data_classification]}
+                                        {:id         "cat"
+                                         :title      "By Category"
+                                         :categories [:data_classification]}]}])}]
      [b/button
       {:icon     "heatmap"
        :text     "Management Regions Layers"
-       :on-click #(re-frame/dispatch [:drawer-panel-stack/push :drawer-panel/management-layers])}]
+       :on-click #(re-frame/dispatch [:drawer-panel-stack/push :drawer-panel/catalogue-layers
+                                      {:group :boundaries
+                                       :title "Management Regions Layers"
+                                       :tabs
+                                       [{:id         "org"
+                                         :title      "By Organisation"
+                                         :categories [:organisation :data_classification]}
+                                        {:id         "cat"
+                                         :title      "By Category"
+                                         :categories [:data_classification]}]}])}]
      [b/button
       {:icon     "more"
        :text     "Third-Party Layers"
-       :on-click #(re-frame/dispatch [:drawer-panel-stack/push :drawer-panel/thirdparty-layers])}]]
+       :on-click #(re-frame/dispatch [:drawer-panel-stack/push :drawer-panel/catalogue-layers
+                                      {:group :third-party
+                                       :title "Third-Party Layers"
+                                       :tabs
+                                       [{:id         "org"
+                                         :title      "By Organisation"
+                                         :categories [:organisation :data_classification]}
```

Comment:
Small thing, but is it feasible to make :data_classification into :data-classification?  (it depends on where it comes from, obviously)

Reply:
It comes from the model:
![Screen Shot 2022-05-31 at 5 31 46 pm](https://user-images.githubusercontent.com/50224230/171117626-fde86dcd-6780-4490-a56e-2ce5b01c42a6.png)
which in turn comes from the DB:
![Screen Shot 2022-05-31 at 5 32 10 pm](https://user-images.githubusercontent.com/50224230/171117708-7fd7165e-4d95-479d-a488-70b8c6cddeb7.png)

So we _could_ change it by massaging the data, but I'm not sure if we want to?